MantidPlot/src/nrutil.cpp

```diff
@@ -5,24 +5,28 @@
 
 #include "nrutil.h"
 
+#include "MantidKernel/Logger.h"
+
+namespace {
+Mantid::Kernel::Logger &LOGGER() {
+  static Mantid::Kernel::Logger glog("nrutil");
+  return glog;
+}
+}
+
 int IMIN(int ia, int ib) {
   if (ia <= ib)
     return ia;
   else
     return ib;
 }
 
-void nrerror(
-    const char *error_text) { // Numerical Recipes standard error handler
-  fprintf(stderr, "Numerical Recipes run-time error...\n");
-  fprintf(stderr, "%s\n", error_text);
-  fprintf(stderr, "...now exiting to system...\n");
-  exit(1);
-}
+// Numerical Recipes standard error handler
+void nrerror(const char *error_text) { LOGGER().error(error_text); }
```

Comment:
Oh so we had `fprintf`s from the old "good" numerical recipes code. Not really the topic of this PR, but there are several places in Framework (including Kernel and Geometry) where we also have `cout`s and `cerr`s. And we have plenty of commented out `cout`s and `cerr`s which should be removed and or replaced with Mantid logs or `TS_TRACE` or similar.
Maybe we could add a maintenance/easy issue for that?